docs/source/models/causes/neonatal/preterm_birth.rst

```diff
@@ -148,23 +149,47 @@ unit time.
       - The probability that a simulant who was born alive dies from preterm without RDS during the neonatal period
 
 
-Data Tables
-+++++++++++
+Modeling Strategy
++++++++++++++++++
+
+The Preterm Birth submodel requires only the birth-weight- and gestation-age-stratified cause specific mortality rates for preterm birth complications with and without respiratory distress syndrome during the early and late neonatal periods.
+
+Since this is a PAF-of-one cause, the calculation must take into account the "structural zeros" representing no mortality risk for simulants with a gestational age of 37 or more weeks.
+
+The way these CSMRs are used is the same for all subcauses, and therefore is included in the :ref:`Overall Neonatal Disorders Model <2021_cause_neonatal_disorders_mncnh>` page.  This page describes the birth-weight- and gestational-age-specific cause specific mortality rates that are used for this cause on that page, :math:`\text{CSMR}^{\text{preterm with RDS}}_{\text{BW},\text{GA}}` and :math:`\text{CSMR}^{\text{preterm without RDS}}_{\text{BW},\text{GA}}`. In both cases, the formula is:
+
+.. math::
+    \begin{align*}
+    \text{CSMR}^{k}_{\text{BW},\text{GA}}
+    &=
+    \begin{cases}
+    \text{CSMR}\cdot f_k \cdot \text{RR}_{\text{BW},\text{GA}} \cdot Z, & \text{if BW} < 37; \\
+    0, & \text{if BW} \geq 37;
+    \end{cases}
+    \end{align*}
 
-The Preterm Birth model requires only the probability of death (aka "mortality risk") for use
-in the decision graph. This will be computed from the overall neonatal mortality risk and the cause-specific mortality fraction.
+where :math:`k` is the subcause of interest (preterm birth with or without RDS),
+:math:`\text{CSMR}` is the cause-specific mortality rate for preterm birth complications,
+:math:`f_k` is the fraction of preterm deaths due to subsubcause :math:`k` (with or without RDS), :math:`\text{RR}_{\text{BW},\text{GA}}` is the relative risk of all-cause mortality for a birth weight of :math:`\text{BW}` and gestational age of :math:`\text{GA}`, and :math:`Z` is a normalizing constant selected so that :math:`\int_{\text{BW<37}} \int_{\text{GA}} \text{RR}_{\text{BW},\text{GA}} \cdot Z = 1`.
```

Comment:
Maybe it would be more transparent to just call this $f\_\text{RDS}$ instead of $f\_k$?

Reply:
Oh, I see. In the formula above, you're using $k$ to parameterize both subcauses, whereas in the data table below, $f\_k$ refers only to the fraction _with_ RDS, which is inconsistent with the usage here.